rpc/eth_api.go

```diff
@@ -115,7 +115,12 @@ func (e *PublicEthAPI) GetBlockTransactionCountByHash(hash common.Hash) hexutil.
 
 // GetBlockTransactionCountByNumber returns the number of transactions in the block identified by number.
 func (e *PublicEthAPI) GetBlockTransactionCountByNumber(blockNum rpc.BlockNumber) hexutil.Uint {
-	return 0
+	node, _ := e.cliCtx.GetNode()
+
+	height := blockNum.Int64()
+	block, _ := node.Block(&height)
```

Comment:
I want to say this will fail if a block number greater than the current block number is passed (can't confirm rn) and then we will get a panic on the next line

Reply:
yeah I'll add error handling for this. I was trying to follow the eth spec closely but it doesn't make sense why error handling doesn't exist for this